File: ympa_fsharp_functions_server/src/model/OfferMappingSuggestionsListDTO.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.EnrichedMappingsOfferDTO

module OfferMappingSuggestionsListDTO =

  //#region OfferMappingSuggestionsListDTO

  [<CLIMutable>]
  type OfferMappingSuggestionsListDTO = {
    [<JsonProperty(PropertyName = "offers")>]
    Offers : EnrichedMappingsOfferDTO[];
  }

  //#endregion

File: src/main/scala/org/openapitools/server/model/OfferMappingSuggestionsListDTO.scala
package org.openapitools.server.model


/**
 * Список рекомендованных карточек товара.
 *
 * @param offers Список товаров. for example: ''null''
*/
final case class OfferMappingSuggestionsListDTO (
  offers: Seq[EnrichedMappingsOfferDTO]
)

